Gopkg.toml

```diff
@@ -43,24 +43,24 @@
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/operatorkit/pull/319 gets merged.